core/src/main/java/com/linecorp/armeria/internal/common/logging/LoggingDecorators.java

```diff
@@ -126,7 +126,9 @@ public static void logResponse(
 
         if (responseLogLevel.isEnabled(logger)) {
             final RequestContext ctx = log.context();
-            if (!log.responseHeaders().status().isServerError() && isTransientService(ctx)) {
+            if (responseCause == null &&
```

Comment:
Should we apply the same approach to request-side logs if `requestCause` is not null?

Reply:
We currently do not log any requests of a transient service unless the option is specified.
https://github.com/line/armeria/blob/master/core/src/main/java/com/linecorp/armeria/internal/common/logging/LoggingDecorators.java#L55

Reply:
So IIUC, I thought we need to change the default behavior of request-side. 
Let's say a service is a `TransientService`. 
- `responseCause != null`, a response will be logged. (By this PR)
- `requestCause != null`, a request will not be logged.
  - Don't we need to log the request if a `requestCause` is not null?

Reply:
I got it now. Yeah, that's a good idea. 😄